k8s/deployment.yml

```diff
@@ -32,6 +32,14 @@ spec:
         - name: "gnupg"
           hostPath:
             path: "/srv/dashboard/storage/.gnupg"
+        - name: "mumble-cert"
+          readOnly: true
+          secret:
+            secretName: "mumble.mafiasi.de-cert"
+        - name: "jabber-cert"
+          readOnly: true
+          secret:
+            secretName: "jabber.mafiasi.de-cert"
```

Comment:
Strictly speaking we only need the certificate and not the key so you could restrict this to only mount the certificate.

But since we are essentially in a trusted environment, you can leave it as si.

Reply:
You are right, I adapted the config to include only the certificate.